Fixes #24481 - update log level on plugin assets

iNecas opened the pull request:
Reasons for changing the log level from warn to debug:

1. the plugin can still register the assets manually

2. in some cases, they even should not be namespaced (seems like it's
needed for example with custom favicon)

We should limit the log messages we propagate to production to just
those that can have influence on the production setup.

iNecas committed:
Fixes #24481 - update log level on plugin assets

Reasons for changing the log level from warn to debug:

1. the plugin can still register the assets manually

2. in some cases, they even should not be namespaced (seems like it's
needed for example with custom favicon)

We should limit the log messages we propagate to production to just
those that can have influence on the production setup.

xprazak2 commented:
@iNecas, test failures are related

iNecas commented:
Thanks, updated

dLobatog reviewed (dismissed):
Seems that the test is still failing @iNecas ? Good to go once :green_apple:

mmoll commented:
@iNecas What's the status here?

iNecas commented:
Rebased and added  a stub so that the expectation doesn't fail on some additional debug logging, that happens to happen during the testing: we're interested only into the assets logging there. The tests are passing locally for me now.

iNecas commented:
foreman tests are green, katello failure doesn't seem to be related in any way. @xprazak2 mind merging?

xprazak2 commented:
@dLobatog, any additional concerns or are we ok to merge?

iNecas commented:
As per https://github.com/theforeman/foreman/pull/5889#pullrequestreview-144010479, seems like green tests was the last thing @dLobatog was requesting, Could we get this merged: I would like to cross this from my TODO list :)

mmoll commented:
merged, díky @iNecas!
